src/twisted/words/im/basesupport.py

```diff
@@ -21,6 +20,8 @@
 
 
 
+# Abstract representation of chat "model" classes
```

Comment:
why a comment and not a docstring?

Reply:
This comment is for multiple classes in this file, not for one specific class.
This comment was in the file before, I just moved it to after the imports.